internal/proto/controller/api/resources/sessions/v1/session.proto

```diff
@@ -49,6 +44,11 @@ message Connection {
 
 // Session contains all fields related to a Session resource
 message Session {
+  // Worker info previously contained only the worker's address.  A session
```

Comment:
This is not quite correct. Note that worker_info is repeated. This is a list of workers that were chosen as allowed proxies at session authorization time. Any connections would need to be made through one of these workers.

IOW, this is the set of workers that matched any worker filter on the target and any load balancing or round robin we might do in the future.

You're right in that _for a particular connection_ you'd have to ask the connection which worker was used. But I'm not convinced that removing this is the right thing to do as it's still potentially valuable information.

Another option might be to change it to something like a list of worker IDs rather than expose the address here directly.